tuplex/adapters/cpython/src/PythonHelpers.cc

```diff
@@ -529,15 +529,15 @@ namespace python {
             if(PyDict_Size(obj) == 0)
                 return Field::empty_dict();
 
-            auto dictType = mapPythonClassToTuplexType(obj);
+            auto dictType = mapPythonClassToTuplexType(obj, true);
```

Comment:
should maybe the function get an autoUpcast parameter as well which is downpassed to the mapPythonClassToTuplexType?